common/src/main/java/org/apache/uniffle/common/security/HadoopSecurityContext.java

```diff
@@ -56,7 +56,7 @@ public HadoopSecurityContext(
 
     if (StringUtils.isNotEmpty(krb5ConfPath)) {
       System.setProperty(KRB5_CONF_KEY, krb5ConfPath);
-      sun.security.krb5.Config.refresh();
+//      sun.security.krb5.Config.refresh();
```

Comment:
Why do we change this line?

Reply:
Sorry, I just commented it to run out, and it has been rolled back.